sympy/stats/joint_rv_types.py

```diff
@@ -177,11 +180,25 @@ def MultivariateT(syms, mu, sigma, v):
     Parameters
     ==========
 
-    syms: list/tuple/set of symbols for identifying each component
-    mu: A list/tuple/set consisting of k means,represents a k
+    sym: A symbol/str for identifying the random variable. This can be
+        indexed to obtain each component of the joint probability space.
```

Comment:
What does this mean the parameter can be indexed? Also, as below, the parameter is still called `syms`. 

Did the API change here or was the documentation just wrong before?

Reply:
The API in the removed documentation was according to a PR that was later closed, so it was just wrong before.
The random variable returned from using these functions can be indexed to return a `Indexed` symbol, with the base as a `JointRandomSymbol`.